2-mandatory/2-financial-times.js

```diff
@@ -5,16 +5,33 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
-}
+  const potentialHeadlines = [];
+
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    const articleTitle = allArticleTitles[i];
+
+    if (articleTitle.length <= 65) {
+      potentialHeadlines.push(articleTitle);
+    }
+  }
 
+  return potentialHeadlines;
+}
+   
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
```

Comment:
Your solution will give you the title with the fewest characters, but this might not be the fewest words.
Can you fix this by making a couple of small changes?